packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -72,6 +72,7 @@ class Answer extends React.Component {
               isOpen={this.state.modalVisible}
               closeModal={this.closeModal}
               query={router.query.q || title}
+              fiche={`${router.route}/${router.query.slug}`}
```

Comment:
ce serait pas plutôt `url` que `fiche` ?

Reply:
`Router.asPath` (string) should work too

Reply:
wondering if `document.location.href` wouldnt be better so we also get hostname :)

Reply:
why not =)